support/templates/agenda-prepublish.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="agendapunten" prefix={{prefixes}}>
+    <span property="http://data.vlaanderen.be/ns/besluit#isGehoudenDoor"
+          resource={{zitting.bestuursorgaan}}> </span>
+    <span property="http://www.w3.org/ns/prov#startedAtTime"
```

Comment:
isGehouden door, startedAtTime etc should be wrapped in a container describing the zitting. e.g.

```html
<div about="http://data.gift/zittingend/xxxx" typeof="besluit:Zitting">
<span property="http://www.w3.org/ns/prov#startedAtTime" ... />
</div>
```